### PR TITLE
Add an Nginx image

### PR DESCRIPTION
**Pull request by andytson-inviqa**
Based on the work from php-nginx with only static file serving from the web root

### CODE REVIEW

**Comment by andytson-inviqa** [comment]
Quay repo has been added with bot user

File: nginx/README.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+The image supports a single server (virtual host) by default:
+
+/etc/nginx/sites-enabled/default
+
+This server configuration is split out into separate component files:
+
+* /etc/nginx/sites-available/default-05-custom_scheme_flags.conf
+* /etc/nginx/sites-available/default-10-base.conf
+* /etc/nginx/sites-available/default-20-rewriteapp.conf
+* /etc/nginx/sites-available/default-30-phpfpm.conf
```
Comment (kierenevans) [inline nginx/README.md]: Can remove this line.

File: nginx/etc/confd/templates/nginx/nginx_site.conf.tmpl [hunk 2]
```diff
@@ -0,0 +1,32 @@
+server {
```
Comment (kierenevans) [inline nginx/etc/confd/templates/nginx/nginx_site.conf.tmpl]: This file should be deleted (no longer written)

Comment (andytson-inviqa) [inline nginx/etc/confd/templates/nginx/nginx_site.conf.tmpl]: nginx_site toml still exists in the conf.d folder

File: nginx/README.md (hunk 1 above)
Comment (andytson-inviqa) [inline nginx/README.md]: removed phpfpm and rewriteapp

**Pull request merged**